src/app/project-memberships/project-membership.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../environments/environment';
+import { Observable } from 'rxjs';
+import { ApproveUserProjectRequest } from './approve-user-project-request';
+import { UserProject } from '../user-projects/user-project';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProjectMembershipService {
+
+  private approveAllUrlPattern = `${environment.serverApiUrl}/users/{userId}/projects`;
+
+  constructor(private httpClient: HttpClient) {
+  }
+
+  approveAll(userId: string, projects: ApproveUserProjectRequest[]): Observable<UserProject[]> {
```

Comment:
I would suggest placing the method `approveAll` in the `UserProjectsService` and naming it `updateUserProjects` since approval of several projects is basically update of the several user projects.
@svetivanova what do you think?